core/trino-parser/src/test/java/io/trino/sql/TestSqlFormatter.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.sql;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.sql.tree.ColumnDefinition;
+import io.trino.sql.tree.CreateTable;
+import io.trino.sql.tree.GenericDataType;
+import io.trino.sql.tree.Identifier;
+import io.trino.sql.tree.NodeLocation;
+import io.trino.sql.tree.QualifiedName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static io.trino.sql.QueryUtil.identifier;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TestSqlFormatter
+{
+    private static final String TABLE_NAME = "table_name";
+    private static final String COLUMN_NAME = "column_name";
+    private static final String RESERVED_TABLE_NAME = "exists";
+    private static final String RESERVED_COLUMN_NAME = "exists";
+    private static final String CREATE_TABLE = "CREATE TABLE %s (\n   %s VARCHAR\n)";
+
+    @Test
+    public void testCreateTable()
+    {
+        assertEquals(String.format(CREATE_TABLE, TABLE_NAME, COLUMN_NAME), createTable(TABLE_NAME, COLUMN_NAME));
```

Comment:
`createTable` is misleading. It's not obvious that it's purpose is to do the bulk of the work -- i.e., formatting a CreateTable node with a specific shape (see last paragraph in this comment).

I would restructure the test as follows (also, using AssertJ assertions):

```java
assertThat(formatSql(createTable(tableName, columnName)))
    .isEqualTo("CREATE TABLE %s (   %s VARCHAR)".formatted(tableName, columnName));
```

and 

```java
private CreateTable createTable(String tableName, String columnName)
{
     ...
     return new CreateTable(...);
}
```

However, I might even just inline the whole createTable method in the caller. There's a tight dependency between what the createTable method does (based on the arbitrary choice of having a single VARCHAR column) and the expected string that comes out of formatting the returned CreateTable node. However, those concerns are scattered throughout the class in the current implementation (static fields, test method, createTable method)

Reply:
- Moved everything into a single method
- Switched to AssertJ
- Kept `String.format` as `trino-parser` is still targeting JDK 8
- Made `createTable` a lambda (Am I overdoing it?)